darwin/Classes/SwiftAudioplayersPlugin.swift

```diff
@@ -696,7 +701,9 @@ public class SwiftAudioplayersPlugin: NSObject, FlutterPlugin {
     }
     
     func stop(playerId: String) {
-        let playerInfo: PlayerInfo = players[playerId]!
+        guard let playerInfo = players[playerId] else {
+            return
+        }
```

Comment:
[Frage] Ist dieser Check hier denn wirklich notwendig? In der `handle` Methode wird doch mit Aufruf von `initPlayerInfo` doch immer sichergestellt, dass der Player mit der `playerId` existiert.

Reply:
Ich hatte etwas aufgeräumt und ein paar häßliche forced unwraps entfernt.

Reply:
Aber die forced unwraps wären an der Stelle doch ok, oder? Ich würde sie persönlich sogar schöner finden. Vor allem denke ich, dass der Autor des Plugins das bewusst so geschrieben hat und es daher unnötige Bedenken bei ihm auslösen könnten, wenn wir unsere eventuell sinnvollen Anpassungen zu dem frühzeitigen Invalidieren der KVO Observer über einen PR auf das Root Repository zurückführen wollen.

Reply:
Wenn man sich den Swift-Code des Plugins allgemein anschaut glaube ich nicht, dass der Author lange Erfahrung mit Swift hat und dies absichtlich entschieden hat. Aber ich kann es zurück nehmen. Da das Plugin mit 0.17 refactored wurde glaube ich aber auch nicht, dass wir von dieser Abzweigung einen PR erstellen würden.